lib/collective-sections.js

```diff
@@ -160,7 +160,7 @@ const showContributeSection = (collective, isAdmin) => {
     case 'AVAILABLE':
       return true;
     case 'DISABLED':
-      return collective.type === CollectiveType.COLLECTIVE && isAdmin;
+      return (collective.type === CollectiveType.COLLECTIVE || collective.type === CollectiveType.EVENT) && isAdmin;
```

Comment:
`FUND` and `PROJECT` also.

Maybe even all `type`s in the future, but not for now.